PySDM/backends/impl_thrust_rtc/methods/collisions_methods.py

```diff
@@ -481,10 +481,10 @@ def __init__(self):
         )
 
         self.__normalize_body_1 = trtc.For(
-            param_names=("prob", "cell_id", "norm_factor"),
+            param_names=("prob", "cell_idx", "cell_id", "norm_factor"),
```

Comment:
@edejong-caltech let's add a minimal test for this one!

Reply:
Perhaps you can help suggest a test @slayoo ?